src/util.js

```diff
@@ -835,3 +835,22 @@ export function createCommentTextNode(annotationText) {
 
     return textEl;
 }
+
+/**
+ * Clears the specified canvas context
+ *
+ * @param {HTMLElement} pageEl The DOM element for the current page
+ * @param {HTMLElement} layerClass The annotation canvas layer CSS class
+ * @return {void}
+ */
+export function clearCanvas(pageEl, layerClass) {
+    const annotationLayerEl = pageEl.querySelector(`.${layerClass}`);
```

Comment:
Can you add `canvas` to your query selector? This will avoid grabbing any elements that do not support `getContext()`.

`querySelector('canvas.${layerClass}')`